### DISCUSSION TITLE
Secure Access HTTPS

### DISCUSSION THREAD

**Discussion by NoOneReally2021** [blakeblackshear/frigate#15410] [Config Support] [answered]
Hi, i would like to use push notifications via web push, my understanding is that the notification configuration is only available if Frigate web gui is accessed via secure access method, however i am unable to find documentation on how to have https configured, i am only wanting to test this at home on local network.
Frigate is running in a container on a nas so localhost wont work.
Any help would be appreciated 

Thanks

**Comment by NickM-27** [comment, marked as answer]
You should be able to do it by just connecting on the 8971 port

**Comment by NoOneReally2021** [comment]
> You should be able to do it by just connecting on the 8971 port

Just tried, got below error
![image](https://github.com/user-attachments/assets/2febe1d9-9c08-4a52-a9e6-66e88262668d)

**Reply by NickM-27** [reply]
What is your config

**Reply by NoOneReally2021** [reply]
Never mind i had to use https and the port and it worked
Thanks

**Reply by NoOneReally2021** [reply]
So where is the notification menu in settings? I still cant see it?

**Reply by NickM-27** [reply]
Does your browser show the connection as secure? You may still need to override the browser to consider that page secure

**Reply by NoOneReally2021** [reply]
no, its saying its not secure, is there no way to bypass it?
i never seen a menu hidden due to website not being secure.
what options do i have?

**Reply by NickM-27** [reply]
You can configure your browser to ignore the security, usually in chrome://flags

**Reply by NickM-27** [reply]
Or you can setup a reverse proxy

**Reply by NoOneReally2021** [reply]
Are notifications even available in Frigate 14 or just 15?

**Reply by NickM-27** [reply]
15 only

**Reply by NoOneReally2021** [reply]
alright ill wait for that release than. thanks for your help
